lib/index.d.ts

```diff
@@ -290,6 +290,21 @@ declare namespace Realm.Sync {
         isAdmin: boolean;
     }
 
+    class Credentials {
+        static usernamePassword(username: string, password: string, createUser?: boolean): Credentials;
+        static facebook(token: string): Credentials;
+        static google(token: string): Credentials;
+        static anonymous(): Credentials;
+        static nickname(value: string, isAdmin?: boolean): Credentials;
+        static azureAD(token: string): Credentials;
+        static jwt(token: string): Credentials;
+        static adminToken(token: string): Credentials;
+
+        readonly provider: string;
```

Comment:
Java uses the term `identityProvider`

Reply:
.NET uses `IdentityProvider` as well, Cocoa uses `provider`, but Cocoa's version is internal, so people can't actually access it. I guess, I'll change it to align with Java and .NET